site-content/source/modules/ROOT/pages/blog/Apache-Cassandra-Changelog-12-February-2022.adoc

```diff
@@ -0,0 +1,93 @@
+
+Note: As the docs are not yet updated, the bintray location for Debian users is replaced with the https://apache.jfrog.io/artifactory/cassandra/[ASF's JFrog Artifactory location^].
+
+See the https://cassandra.apache.org/download/[download section] for the latest stable and older supported versions of source and binary distributions.
+
+To stay up-to-date, we recommend joining the Cassandra  https://cassandra.apache.org/community/[mailing list].
+
+== Community Notes
+
+_Updates on Cassandra Enhancement Proposals (CEPs), how to contribute, and other community activities._
+
+_Are you new to the project?  We have a handy https://cassandra.apache.org/_/development/index.html[‘Contributing to Cassandra’] page on how to get involved and get started. Additionally, we have established two quick labels you should take a look at if you are new to the project. One is for https://issues.apache.org/jira/secure/RapidBoard.jspa?rapidView=496&quickFilter=2252[“Failing Tests”^] and the other corresponds to our https://issues.apache.org/jira/secure/RapidBoard.jspa?rapidView=484&quickFilter=2162[“Low Hanging Fruit”^]. Feel free to self-select a ticket to work on. Any of these tickets should be of appropriate complexity for someone new to the project to tackle. Just remember to put a note on the ticket. You can also reach out on the https://the-asf.slack.com/[ASF Slack^] in the #cassandra-dev Slack channel. Use @cassandra_mentors to contact any of the 13 designated Cassandra mentors!_
+
+Read PMC member Josh McKenzie’s https://lists.apache.org/thread/pclo1tpqsfkc4skxhftrydos89o1t72q[latest bi-weekly update^] for ongoing discussions and the latest on ticket progress.
+
+=== Added
+
+A massive thank you goes to *Lorina Poland* for converting all the Cassandra documentation from reStructuredText to AsciiDoc for the new website and another heartfelt thank you to *Mick Semb Wever* for getting the ticket over the line by working over the holiday period. We also wouldn’t have the site without *Anthony Grasso’s* extensive work moving us to the https://antora.org/[Antora^] platform. Thank you all! 👏
+
+:!figure-caption:
+
+.Take a look at the newly updated Cassandra documentation site.
+[link=https://cassandra.apache.org/doc/latest/]
+image::blog/cassandra-documentation-changelog-12.png[New Cassandra documentation website]
+
+=== Added
+
+Not sure what branch to target with your work? *Josh McKenzie* has put together a https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=199530302[wiki page^]
+
+=== Added
+
+We have established a Build Lead role and we are seeking more volunteers. The position is inspired by the ‘Build Baron’ role used in MongoDB, but more focused on triaging test failures and database correctness, but it may evolve into a performance regression and change point triage role in the future. Read the https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=199527692[wiki page^] for what’s involved in this rotating position and sign-up!
```

Comment:
```suggestion
We have established a Build Lead role and we are seeking more volunteers. The position is inspired by the ‘Build Baron’ role used in MongoDB, but more focused on triaging test failures and database correctness, but it may evolve into a performance regression and change point triage role in the future. Read the https://cwiki.apache.org/confluence/x/DI3kCw[wiki page^] for what’s involved in this rotating position and sign-up!
```